newIDE/app/public/external/utils/path-editor.js

```diff
@@ -97,6 +152,10 @@ const render = headerObject => {
   headerObject.nameInput.style.width =
     (headerObject.nameInput.value.length + 1) * 10 + 'px';
   // check if it will overwrite a file and notify the user in a subtle, but obvious way
+  // but don't do it if there is no extension (image sequence will be saved)
+  if (headerObject.state.extension === '-') {
```

Comment:
I have the feeling that this magic value "-" is used to store more information that just the extension, as you added a comment "dont do it if there is no extension (image sequence will be saved)".

I'm not sure to understand what's the link between "there is no extension" and "image sequence will be saved"? Are you sure you're not using the extension as a "hack" to store something that should be in a separate variable, in the state or options?

Reply:
Yes in this case I used '-' to determine that no extension is going to be used by the path editor. An alternative is to use nothing or undefined instead? Or I can create a new parameter?

Reply:
I was, you're right. Will update it :)